vermouth/dssp/dssp.py

```diff
@@ -145,7 +153,67 @@ def read_dssp2(lines):
     return secstructs
 
 
-def run_dssp(system, executable='dssp', savefile=None, defer_writing=True):
+def run_mdtraj(system):
+    """
+    Compute DSSP secondary structure assignments for the system by using
+    ``mdtraj.compute_dssp``.
+
+    During processing, a PDB file is produced. Therefore, all the molecules
+    in the system must contain the required attributes for such a file to be
+    generated. Also, the atom names are assumed to be compatible with the
+    'charmm' force field for MDTraj to recognize them.
```

Comment:
At least it's true-ish for dssp, you can't use just any atomname. I didn't check how what where with mdtraj, but since it uses it's own bundled copy of dssp I assume it behaves the same. 

I really don't know if it's limited to charmm or some odd subset of atomistic force fields, or just the HNCO atoms, and honestly I don't care enough to read the dssp source

Reply:
ok!